### PR TITLE
Update nakadi-event-bus-api.yaml

### PR DESCRIPTION
**Pull request by lmontrieux**
Fix documentation

### CODE REVIEW

**Comment by lmontrieux** [comment]
👍

File: docs/_data/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -828,7 +828,7 @@ paths:
               The flow id of the request, which is written into the logs and passed to called services. Helpful
               for operational troubleshooting and log analysis.
             type: string
-          - name: cursors
+          - name: cursors/
```
Comment (v-stepanov) [inline docs/_data/nakadi-event-bus-api.yaml]: why "cursors/" ?

Comment (lmontrieux) [reply docs/_data/nakadi-event-bus-api.yaml]: I don't know how I changed this. Will revert

Comment (lmontrieux) [reply docs/_data/nakadi-event-bus-api.yaml]: reverted.

**Comment by lmontrieux** [comment]
:+1:

**Comment by rcillo** [comment]
👍

**Pull request merged**